submissions/ASTkach/popup/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link href="https://fonts.googleapis.com/css2?family=Coiny&family=Merriweather&display=swap" rel="stylesheet" />
+        <link rel="stylesheet" href="css/app.css" />
+        <title>Hooli-style Popup</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header" id="home">
+                <div class="header__container">
+                    <a href="#home" class="header__logo" tabindex="1">Cat.</a>
+                    <nav class="nav">
+                        <ul class="nav__list">
+                            <li class="nav__item nav__item--hidden">
+                                <a class="nav__link" href="#" target="_blank" rel="nofollow noopener" tabindex="2">Mail</a>
+                            </li>
+                            <li class="nav__item nav__item--hidden">
+                                <a class="nav__link" href="#" target="_blank" rel="nofollow noopener" tabindex="3">Photo</a>
+                            </li>
+                            <li class="nav__item">
+                                <div class="nav__popup _icon-popup-button">
```

Comment:
When the popup is open focus must go to the first interactive element inside the popup. In our case when the user opens popup and presses the` Tab` key focus must go to the first link in the popup